ui/src/config.js

```diff
@@ -36,6 +36,11 @@ export const appConfig = {
     : [{ href: "https://github.com/caraml-dev/xp", label: "XP User Guide" }],
   pagination: {
     defaultPageSize: 10,
+    experimentContextPageSize: 50,
+  },
+  listExperimentFields: {
```

Comment:
I'm not sure this should be an app-level config. Can it just be embedded in the components that use them? Reasons:
* If this were an app-config, I think it would make sense to let it drive the behavior of the components entirely (like modifying the `experimentTableFields` here could also change the columns displayed in the experiments table, which is not the case now and I think that would be over-engineering at this point).
* These configs are not re-used in multiple places which could also be a hint that this can be localised.

If either of the above reasons changed, we could consider extracting them out again but perhaps we will need to think more about what influence such configs will have (bullet # 1 above).

Reply:
That's true, I was thinking of #1 as a distant possibility but yeah now that you mention it, it's probably over-engineering; I've removed both configs and directly embedded the lists in the components they are used.